### PR TITLE
Issue #3258167 by agami4: Add route condition, add class and library

### PR DESCRIPTION
**Pull request by agami4**
## Problem
The remove/delete membership group page does not have a correct display.

## Solution
Add class to the membership group page and attach library.

## Issue tracker
https://getopensocial.atlassian.net/browse/YANG-6904
https://www.drupal.org/project/social/issues/3258167

## How to test
*For example*
- [x] Create/Go to group/challenge page.
- [x] Go to the `Manage participants`/`Manage members` page.
- [x] Click on the `Actions` dropdown button on the user-item.
- [x] Click on Edit/Delete button.

## Screenshots
before:
Screenshot 2022-01-12 at 12 05 59
Screenshot 2022-01-12 at 12 06 11

after:
Screenshot 2022-01-12 at 13 02 23
Screenshot 2022-01-12 at 13 02 30

## Release notes
The edit/delete group membership page has a correct layout.

## Change Record
N/A

## Translations
N/A

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2698-joegz6hksp1qyh9lwnxj1bwspgfvbgg6.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61e58bb7f67b7c778cfd4f5b

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2698-joegz6hksp1qyh9lwnxj1bwspgfvbgg6.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61e58bb7f67b7c778cfd4f5b

**Comment by Ressinel** [comment]
@agami4 I see this result on the Tugboat domain.

[Edit](https://pr2698-joegz6hksp1qyh9lwnxj1bwspgfvbgg6.tugboat.qa/group/5/content/19/edit):

edit

[Remove](https://pr2698-joegz6hksp1qyh9lwnxj1bwspgfvbgg6.tugboat.qa/group/5/content/19/delete):

remove

**Comment by agami4** [comment]
@Ressinel, You tested changes without theme changes

**Comment by Ressinel** [comment]
@agami4 Could you add to the HTT section which theme changes I should implement before testing? Since there is no any notices about the theme patch.

**Comment by agami4** [comment]
@Ressinel, you can check fixes on those pages:

- https://x-bitbucket-pr-313-oogl52y-vncdokyquw6mc.eu.platform.sh/group/54/content/152/edit?destination=/group/all-visibility-options/membership
- https://x-bitbucket-pr-313-oogl52y-vncdokyquw6mc.eu.platform.sh/group/54/content/152/delete?destination=/group/all-visibility-options/membership
- https://x-bitbucket-pr-313-oogl52y-vncdokyquw6mc.eu.platform.sh/group/54/member/152

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2698-joegz6hksp1qyh9lwnxj1bwspgfvbgg6.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61e58bb7f67b7c778cfd4f5b

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2698-joegz6hksp1qyh9lwnxj1bwspgfvbgg6.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61e58bb7f67b7c778cfd4f5b

**Comment by agami4** [comment]
@Ressinel, I've added new route to the condition and you can check new changes here:
https://x-bitbucket-pr-313-oogl52y-vncdokyquw6mc.eu.platform.sh/group/all-visibility-options/translations

File: modules/social_features/social_group/social_group.module [hunk 1]
```diff
@@ -2805,3 +2805,15 @@ function social_group_preprocess_page(array &$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_html().
+ */
+function social_group_preprocess_html(array &$variables):void {
```
Comment (Ressinel) [inline modules/social_features/social_group/social_group.module:2812]: @agami4 Let's stick to one style and add a space before the `void`:
`function social_group_preprocess_html(array &$variables): void {`

**Review by Ressinel — CHANGES_REQUESTED** [review]

File: modules/social_features/social_group/social_group.module (hunk 1 above)
Comment (agami4) [reply modules/social_features/social_group/social_group.module]: @Ressinel done

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2698-joegz6hksp1qyh9lwnxj1bwspgfvbgg6.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61e58bb7f67b7c778cfd4f5b

**Comment by Ressinel** [comment]
@agami4 Couldn't we add this preprocess to the **socialblue** theme directly? Namely in the file `socialblue.theme`.

**Comment by agami4** [comment]
Move changes to socialblue theme

**Comment by Ressinel** [comment]
Appropriate PR in the socialblue theme:
https://git.drupalcode.org/project/socialblue/-/merge_requests/60

**Pull request closed without merging**